vip-jetpack/vip-jetpack.php

```diff
@@ -413,3 +413,18 @@ function add_jetpack_menu_placeholder(): void {
 }
 
 add_action( 'admin_menu', 'add_jetpack_menu_placeholder', 999 );
+
+/**
+ * Remove the marketing admin page for Jetpack Search if VIP Search is enabled
+ */
+function remove_jetpack_menu_search(): void {
+	if ( defined( 'VIP_ENABLE_VIP_SEARCH' ) && true === VIP_ENABLE_VIP_SEARCH ) {
+		if ( class_exists( 'Jetpack_Search_Dashboard_Page' ) ) {
+			remove_submenu_page(
+				'jetpack',
+				'jetpack-search',
+			);
+		}
+	}
+}
+add_action( 'admin_menu', 'remove_jetpack_menu_search', 999 );
```

Comment:
Awesome. Should we move here though ? - https://github.com/Automattic/vip-go-mu-plugins/blob/ab2e84eba1e548a518a53bf8c8a2ee6b394fd89c/search/search.php#L30